sql/moz-fx-data-shared-prod/fenix/firefox_android_clients/view.sql

```diff
@@ -33,6 +29,6 @@ SELECT
     WHEN install_source NOT IN ('com.android.vending')
       THEN 'Other'
     ELSE install_source
-  END AS install_source,
+  END AS install_source
```

Comment:
```suggestion
  END AS install_source,
```
nit: keep the trailing comma